### PR TITLE
Update zkEVM - the ulxly

### PR DESCRIPTION
**Pull request by EmpieichO**
Update zkEVM - Adding the ulxly bridge doc.

### CODE REVIEW

File: docs/zkEVM/architecture/protocol/unified-LxLy/ulxly-rollupmanager.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+Unified LxLy bridge is a more robust version of the LxLy bridge as it is designed for adding chains to the Polygon ecosystem, and thus allow interoperability of many chains, both L2 and L1 networks.
```
Comment (hsutaiyu) [inline docs/zkEVM/architecture/protocol/unified-LxLy/ulxly-rollupmanager.md]:
```suggestion
Unified bridge is a more robust version of the LxLy bridge as it is designed for adding chains to the Polygon ecosystem, and thus allow interoperability of many chains, both L2 and L1 networks.
```

File: mkdocs.yml [hunk 2]
```diff
@@ -154,7 +154,13 @@ nav:
             - Exit trees: zkEVM/architecture/protocol/zkevm-bridge/exit-tree.md
             - Smart contracts: zkEVM/architecture/protocol/zkevm-bridge/smart-contracts.md
             - Flow of assets: zkEVM/architecture/protocol/zkevm-bridge/flow-of-assets.md
-          - LXLY bridge: zkEVM/architecture/protocol/lxly-bridge.md
+          - LxLy bridge: zkEVM/architecture/protocol/lxly-bridge.md
+          - Unified LxLy bridge:
```
Comment (hsutaiyu) [inline mkdocs.yml]:
```suggestion
          - Unified bridge:
```

**Review by hsutaiyu — APPROVED** [review]
Suggest not using LaTeX for code keywords such as function names, interface names, structure names, etc. since it takes much longer to render and isn't necessary.

File: docs/zkEVM/architecture/protocol/unified-LxLy/ulxly-rollupmanager.md (hunk 1 above)
Comment (hsutaiyu) [reply docs/zkEVM/architecture/protocol/unified-LxLy/ulxly-rollupmanager.md]: Also, "Unified bridge is a more robust version of the LxLy bridge" may not be the best way to describe the unified bridge. It isn't necessarily "better" in any way. Just meant to enable a different set of use cases (staking/restaking, and control of assets locked in the escrow contract). Won't necessarily be used by all chains either.

Comment (EmpieichO) [reply docs/zkEVM/architecture/protocol/unified-LxLy/ulxly-rollupmanager.md]: After reviewing your PR#332 ... I have a better understanding of the unified bridge. The source of content I have simply states facts and does not provide sufficient context.

**Pull request merged**